src/components/Tooltip/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './ToolTip';
```

Comment:
wrong file name?
export { default } from './tooltip;?

Reply:
It is right. I changed the filename to be consistent

Reply:
Yes, in that case, can you update the tooltip.js file name to ToolTip.js. Otherwise, it will throw an error saying file name not found

Reply:
![image](https://user-images.githubusercontent.com/26462504/121379186-cadc6300-c911-11eb-94e2-c9d628a223a2.png)

Reply:
Fixed